pages.it/common/kubectl.md

```diff
@@ -0,0 +1,37 @@
+# kubectl
+
+> nterfaccia interattiva da linea di comando per eseguire comandi sui clusters Kubernetes.
+> Alcuni sottocomandi, come `kubectl run` hanno la loro documentazione per l'uso.
```

Comment:
We have this standard translation https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#it
```suggestion
> Alcuni comandi aggiuntivi, come `kubectl run`, hanno la propria documentazione..
```